examples/gallery/demos/bokeh/autompg_histogram.ipynb

```diff
@@ -53,8 +53,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "opts.defaults(opts.Histogram(alpha=0.9, width=600))\n",
-    "autompg_ds.hist(dimension='mpg', groupby='cyl', bin_range=(9, 46), bins=40, adjoin=False)"
+    "hist = autompg_ds.hist(dimension='mpg', groupby='cyl', bin_range=(9, 46), bins=40, adjoin=False)\n",
```

Comment:
I wasn't sure about this one and nearly did it the way you did it here. The issue is that the line is getting too long...

Reply:
I find using defaults for a single plot a bit weird personally.

Reply:
Right. That is why I hesitated and don't mind this version.